data_model/1.3/clusters/DoorLock.xml

```diff
@@ -570,7 +570,7 @@ Davis, CA 95616, USA
       <bitfield name="LockFactoryReset" bit="1" summary="Lock Reset to Factory Defaults">
         <optionalConform/>
       </bitfield>
-      <bitfield name="N" bit="2" summary="Reserved">
+      <bitfield name="N/A" bit="2" summary="Reserved">
```

Comment:
Hmm, this is properly a name we should adjust in the spec. Noted.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10079

I assigned both of us, but feel free to remove yourself. Added the DM label.